packages/workflow/src/internals.ts

```diff
@@ -416,9 +425,56 @@ export class State {
     [
       '__enhanced_stack_trace',
       (): EnhancedStackTrace => {
-        // const stacks = this.getStackTraces();
-        // const sourceMap = this.sourceMap;
-        return { locations: [], sources: {} };
+        const sourceMap = this.sourceMap;
+        const stacks = this.getStackTraces().join('\n\n');
```

Comment:
Why join?

Reply:
stacks is an array that should map to a [`StackTrace`](https://github.com/temporalio/proposals/pull/61/files#diff-d9cade3a12b6c56306dfe187736032b0cc8d5b2eed522d605155391dfe62f5f0R65) object

Reply:
When you fix this it'll change the implementation of this method.

Reply:
I had to call join() on the StackTrace because the Error constructor takes stacks as a string.
`const parsedStacks = errorStackParser.parse({ name: '', message: '', stack: stacks });`

Reply:
You need to parse each individual stack not join them all together into a single stack.